webapp/channels/src/components/setting_item.tsx

```diff
@@ -41,37 +41,42 @@ type Props = {
     collapsedEditButtonWhenDisabled?: ReactNode;
 }
 
-export default class SettingItem extends React.PureComponent<Props> {
-    minRef: RefObject<SettingItemMinComponent>;
-
-    constructor(props: Props) {
-        super(props);
-
-        this.minRef = React.createRef();
-    }
+const SettingItem: FC<Props> = ({
```

Comment:
Any strong reason to be explicit about the type? In general is not needed and we don't add it, so curious if you have any strong opinion about it.

Reply:
No reason. I can change the component to be this 
```
{
    active,
    areAllSectionsInactive,
    section,
    max,
    updateSection,
    title,
    isDisabled,
    describe = '',
    collapsedEditButtonWhenDisabled = false,
}: Props
```

Reply:
Yeah, just to be uniform with other components, let's remove the typing.

Reply:
It has been removed